apps/arena/lib/arena/game_updater.ex

```diff
@@ -821,6 +859,20 @@ defmodule Arena.GameUpdater do
     %{game_state | players: updated_players}
   end
 
+  defp handle_destroyed_crates(%{crates: crates} = game_state, game_config) do
+    Enum.reduce(crates, game_state, fn {crate_id, crate}, game_state ->
+      if Crate.alive?(crate) or crate.aditional_info.status == :DESTROYED do
```

Comment:
Is this removing the crates? Or are we not supposed to?

Reply:
nope, this is not removing the crates from the game, just checking if it should spawn a power up and destroying it. Would you rather remove destroyed crates from the state?

Reply:
Aaahh got it, honestly no strong opinions about it. We do have problems with the size of the binary, but a couple crates is the least of our problems and we can tackle it later on